NEWS.md

```diff
@@ -2,6 +2,10 @@
 
 * `first2()`, a `fct_reorder2()` helper function, sorts `.y` by the first value of `.x` (@jtr13).
 
+## Minor bug fixes and improvements
```

Comment:
Since this is just a documentation improvement (and not a user facing change), it doesn't need a news bullet.

Reply:
Otherwise, looks great!

Reply:
I removed the news bullet.